src/pages/Stream/Views/Manage/Info.tsx

```diff
@@ -3,10 +3,9 @@ import classes from '../../styles/Management.module.css';
 import { useStreamStore } from '../../providers/StreamProvider';
 import _ from 'lodash';
 import { useAppStore } from '@/layouts/MainLayout/providers/AppProvider';
-import UpdateTimePartitionLimit from './UpdateTimePartitionLimit';
-import UpdateCustomPartitionField from './UpdateCustomPartitionField';
 import timeRangeUtils from '@/utils/timeRangeUtils';
 import ErrorView from './ErrorView';
+import PartitionLimit from './PartitionLimit';
```

Comment:
This Component does not indicate that it is updating the CustomPartition Field also. A better title for the component is required

Reply:
@Koustavd18 It's a generic component now, so some other alternatives maybe `PartitionFieldLimit`, `PartitionConfig` or `PartitionFieldSelector`

Reply:
How About UpdateStreamInfo

Reply:
Changed it